internal/service/pushbasedlogexport/resource_migration_test.go

```diff
@@ -0,0 +1,68 @@
+package pushbasedlogexport_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
+)
+
+func TestMigPushBasedLogExport_basic(t *testing.T) {
+	mig.SkipIfVersionBelow(t, "1.16.0")
+
+	var (
+		orgID                = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName          = acc.RandomProjectName()
```

Comment:
can we use ProjectIDExecution to pass the project id? or it's a global resource to the project that can cause problems?

Reply:
yeah this resource is configured per project so we can't use ProjectIDExecution

Reply:
does each test take very long?  it's ok as it is, or we can use ProjectIDExecution and run in serial if it doesn't take a lot of time

Reply:
each test doesn't take more than a minute or two I think

Reply:
in that case I would explore the use of ProjectIDExecution with resource.Test instead of ParallelTest

Reply:
updated to use ProjectIDExecution with tests in serial